### DISCUSSION TITLE
Add open-vm-tools package to Deepin V23 daily build

### DISCUSSION THREAD

**Discussion by vampywiz17** [linuxdeepin/developer-center#3551] [Features Request & Ideas | 特性请求 & 头脑风暴]
Hello there, 

if possible, please add open-vm-tools and open-vm-tools-desktop packages (and dependencies) to V23 daily build. It help a lot to testing!

**Comment by vampywiz17** [comment]
[open-vm-tools.zip](https://github.com/linuxdeepin/developer-center/files/10537273/open-vm-tools.zip)

It is the sid/bookworm .deb backages that need it. Only unzip it and run this command (in this folder):

`sudo apt install ./open-vm* ./lib*`

**Reply by xzl01** [reply]
cc @Zeno-sole
